pkg/history/block_validator.go

```diff
@@ -17,7 +17,7 @@
 package history
 
 import (
-	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
```

Comment:
What's the purpose of aliasing the import? I'm a fan or using the default import unless there are collisions, but happy to reconsider

Reply:
Hmm not sure, I think I just noticed us using aliases in other files, but maybe there were collisions in those.